packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+# 47.17.1 - January 30, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-apex
+
+- Update Apex Language Server error message to include we support Java 8 and Java 11 ([PR #1900](https://github.com/forcedotcom/salesforcedx-vscode/pull/1900))
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Set cursor on the first argument when inserting method from code completion ([PR #1913](https://github.com/forcedotcom/salesforcedx-vscode/pull/1913))
+  ![GIF showing Apex code completion with new tab fix](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode/images/47.17.1/autocomplete_tabs.gif)
+
+- Disable Global Objects from autocompletion when invoked from a member type ([PR #1913](https://github.com/forcedotcom/salesforcedx-vscode/pull/1913))
+  ![GIF showing autocompletion results without global objects](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode/images/47.17.1/autocomplete_no_global_objects.gif)
+
+- Show documentation for autocomplete items ([PR #1913](https://github.com/forcedotcom/salesforcedx-vscode/pull/1913))
```

Comment:
```suggestion
- Show documentation for autocomplete items when definitions have block comments ([PR #1913](https://github.com/forcedotcom/salesforcedx-vscode/pull/1913))
```

Want to make clear it's currently just for things the user has added comments for at the moment